llvm/tools/llvm-exegesis/lib/Assembler.cpp

```diff
@@ -168,7 +168,15 @@ void BasicBlockFiller::addInstruction(const MCInst &Inst, const DebugLoc &DL) {
     } else if (Op.isImm()) {
       Builder.addImm(Op.getImm());
     } else if (!Op.isValid()) {
-      llvm_unreachable("Operand is not set");
+      std::string Message;
```

Comment:
This addition here looks unrelated (directly at least) to the RISCV support? Maybe pull this out into a separate patch?

Reply:
I agree with you, thanks. Addressed